src/app/space/create/deployments/apps/collapsible-deployment-info.component.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Component,
+  Input,
+} from '@angular/core';
+
+import { Observable } from 'rxjs';
+
+import { Environment } from '../models/environment';
+
+@Component({
+  selector: 'collapsible-deployment-info',
+  templateUrl: 'collapsible-deployment-info.component.html'
+})
+export class CollapsibleDeploymentInfoComponent {
+
+  static chartIdNum = 0;
+
+  @Input() collapsed: boolean;
+  @Input() applicationId: string;
+
+  public config: any = {
+    chartHeight: 60,
+    chartId: 'chart-' + (CollapsibleDeploymentInfoComponent.chartIdNum++) + '-'
```

Comment:
Please do `chartId: uniqueId('deployment-detail-chart-') + '-'` (https://lodash.com/docs/4.17.4#uniqueId) rather than using this static counter.

Looks good otherwise.